README.md

```diff
@@ -337,6 +337,31 @@ const NewBookForm = () => {
 };
 ```
 
+### Abort Signal
```

Comment:
Im not sure that this should be documented in this mannor, else the readme should document all of the fetch apis. 
Maybe this should be framed as an example of aborting an inflight request on unmount.